XiEditor/EditViewController.swift

```diff
@@ -32,9 +32,20 @@ class EditViewController: NSViewController, EditViewDataSource {
     @IBOutlet weak var editContainerView: EditContainerView!
     @IBOutlet var editView: EditView!
     @IBOutlet weak var gutterView: GutterView!
+    @IBOutlet weak var stackView: NSStackView!
```

Comment:
On further review, I can offer more concrete criticism: by using a stack view, showing the find window causes the `EditView` to move, which moves the text. I would prefer (ultimately/one day) that the find view be drawn _over_ the `EditView`. But no need to do this now.